elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -58,6 +66,9 @@
     private final MultipleFilterDialect filterDialect;
     private final Map<String, FilterExpression> expressionsByType;
 
+    @Getter private final Map<String, Long> checkStats;
+    @Getter private final Set<Triple<Class, Class, String>> expressionResultShortCircuit;
```

Comment:
I'm wondering if this belongs here since all we have is a getter.  I'm not sure where it ought to go.  Would it make more sense in the ActivePermissionExecutor?